GIT_CONFLICTS.md

```diff
@@ -0,0 +1,77 @@
+# Wanderlust Project
+
+## Overview
+Welcome to the Wanderlust project! This is a collaborative effort to build an amazing application. We appreciate your contributions and aim to make the process as smooth as possible.
+
+## Common QnA for Git and Open Source Contributions
+
+<details>
+<summary>
+
+### How can I know if I’m in a bad situation?
+</summary>

```

Comment:
Meaning full heading please!

"bad situation" can be anything this solution might not be the solution for someone some other bad situation, so doesn't actually say perfectly why i have to see this section

Reply:
done